### ISSUE TITLE
Summon Vision not functioning as expected

### ISSUE THREAD

**Issue by NutritiousCookie** [theripper93/Levels#52]
With levels enabled, tokens will disappear if the selected token has no vision. This is an issue for players that control "summoned" tokens (for spells or familiars that get their own spot on the map, but don't allow them to look around corners). However, fog of war functions like it should in vanilla foundry (still tied to the token with vision enabled), but all non-owned tokens disappear and roof visibility is tied to the selected token.

So, to sum it up - when two tokens are controlled, one with vision enabled and one without, selecting the one without vision will cause tokens and roof tiles to appear like they would to the token _without_ vision, but fog of war appears to be drawn as it would be seen by the token _with_ vision. Tokens and roof tiles should be drawn as they would be seen by the token with vision.

**Comment by theripper93** [comment]
should be fixed
